website/docs/for-web/performance/ios/profiling.md

```diff
@@ -0,0 +1,177 @@
+    data-zoom-src={useBaseUrl('/img/profiling/ios/full/profiling-network-01.webP')}
+    width="75%"
+  />
+</em>
+
+## Network
+
+Another tab of interest is "Network". To see the time it takes to load the files from disk, you can click the "Reload" button, press ⌘R, or ⌥⌘R to reload the page ignoring the cache. 
+
+<em style={{ textAlign: 'center', display: 'block' }}>
+  <img 
+    src={useBaseUrl('/img/profiling/ios/thumb/profiling-network-02.webP')} 
+    data-zoom-src={useBaseUrl('/img/profiling/ios/full/profiling-network-02.webP')}
+    width="75%"
+  />
+</em>
+
+Most of the data in the network tab is likely going to be a combination of html, css, and js. The data points for these bits is likely not going to be of much interest since all of these files are loading from disk and should not take long to load. However, if your web team is **_not_** using the CapacitorHttp plugin and any network requests occur in your page interactions, you will see those network requests in this tab as well.
+
+<em style={{ textAlign: 'center', display: 'block' }}>
+  <img 
+    src={useBaseUrl('/img/profiling/ios/thumb/profiling-network-03.webP')} 
+    data-zoom-src={useBaseUrl('/img/profiling/ios/full/profiling-network-03.webP')}
+    width="75%"
+  />
+</em>
+
+## Timeline
+
+One of the more useful features of Safari Dev Tools is the Timeline. Similar to the [Network](#network) tab, you can click the "Reload" button, press ⌘R, or ⌥⌘R to reload the page ignoring the cache to start recording the Timeline. 
+
+<em style={{ textAlign: 'center', display: 'block' }}>
+  <img 
+    src={useBaseUrl('/img/profiling/ios/thumb/profiling-timeline-record-01.webP')} 
+    data-zoom-src={useBaseUrl('/img/profiling/ios/full/profiling-timeline-record-01.webP')}
+    width="75%"
+  />
+</em>
+
+The timeline will continue to record until you click the stop button.
+
+<em style={{ textAlign: 'center', display: 'block' }}>
+  <img 
+    src={useBaseUrl('/img/profiling/ios/thumb/profiling-timeline-record-02.webP')} 
+    data-zoom-src={useBaseUrl('/img/profiling/ios/full/profiling-timeline-record-02.webP')}
+    width="75%"
+  />
+</em>
+
+Once the timeline is finished recording, you can zoom in and slice the parts of the timeline that are of interest.
+
+<em style={{ textAlign: 'center', display: 'block' }}>
+  <img 
+    src={useBaseUrl('/img/profiling/ios/thumb/profiling-timeline-record-03.webP')} 
+    data-zoom-src={useBaseUrl('/img/profiling/ios/full/profiling-timeline-record-03.webP')}
+    width="75%"
+  />
+</em>
+
+### Native Http Caveat
+
+Due to a limitation in Safari Dev Tools, if the web application uses CapacitorHttp you will not get any insight into how those requests may be impacting your application performance in the timeline tab. Starting in Capacitor **insert version here**, a total time for the network request will be logged out to the console. This is not an issue for Android developers debugging their web applications in Chrome.
```

Comment:
we might want to indicate this is in a future version instead of 'insert version here'

Reply:
This is available in 4.6.3 actually